package.json

```diff
@@ -102,6 +110,13 @@
     "storybook:static:stop": "pkill -f \"sh -c serve docs --port 5555\"",
     "visual:test": "backstop test --config config/backstop/config.js",
     "visual:approve": "backstop approve --config config/backstop/config.js",
+    "svg:clean-build": "rimraf build/icons",
+    "svg:clean-components": "del 'src/components/Icon/icons/*.jsx'",
```

Comment:
`del`?

Reply:
deletes individual files -- in this case the previously-generated .jsx react icon components, but not the index.ts file. is there a better way to do this?